community/planning/griddle/openapi.yaml

```diff
@@ -0,0 +1,2146 @@
+        submitter:
+          type: string
+          example: 02cd3181dbd7d1539f470436ce222c53ab5e514f67809dc0095895e6cdfba97612
+        created_at:
+          $ref: "#/components/schemas/Timestamp"
+        order_xml_v3_4:
+          type: string
+          example: "<xml content>"
+        service_id:
+          $ref: "#/components/schemas/ServiceID"
+    PurchaseOrderRevisionList:
+      properties:
+        data:
+          type: array
+          items:
+            $ref: "#/components/schemas/PurchaseOrderRevision"
+        paging:
+          $ref: "#/components/schemas/Paging"
+
+    # Schema models
+    PostSchemas:
+      properties:
+        data:
+          type: array
+          items:
+            $ref: "#/components/schemas/Schema"
+    SchemaList:
+      properties:
+        data:
+          type: array
+          items:
+            $ref: "#/components/schemas/Schema"
+        paging:
+          $ref: "#/components/schemas/Paging"
+    Schema:
+      properties:
+        name:
+          type: string
+          example: Lightbulb
+        description:
+          type: string
+          example: Example Lightbulb schema
+        owner:
+          type: string
+          example: philips001
+        properties:
+          type: array
+          items:
+            $ref: "#/components/schemas/PropertyDefinition"
+        service_id:
+          $ref: "#/components/schemas/ServiceID"
+        last_updated:
+          $ref: "#/components/schemas/Timestamp"
+    PropertyDefinition:
+      properties:
+        name:
+          type: string
+          example: size
+        data_type:
+          $ref: "#/components/schemas/DataTypeEnum"
+        description:
+          type: string
+          example: Lightbulb radius, in millimeters
+        required:
+          type: boolean
+          example: true
+        value:
+          type: string
+          example: 20
+    DataTypeEnum:
+      type: string
+      enum:
+        - STRING
+        - BOOLEAN
+        - NUMBER
+        - BYTES
+        - ENUM
+        - STRUCT
+        - LOCATION
+    PropertyValue:
+      type: object
+      properties:
+        name:
+          type: string
+          example: is_current
+        data_type:
+          type: string
+          example: boolean
+        value:
+          type: string
+          example: false
+    BytesValue:
+      type: string
+      format: byte
+      example: "AQIDBA=="
+    LatLong:
+      type: object
+      properties:
+        latitude:
+          type: integer
+          example: 46786299
+        longitude:
+          type: integer
+          example: -92051336
+    StructPropertyValue:
+      type: object
+      properties:
+        name:
+          type: string
+          example: location
+        type:
+          $ref: "#/components/schemas/DataTypeEnum"
+        value:
+          oneOf:
+            - type: string
+            - type: boolean
+            - type: number
+              format: int64
```

Comment:
`int64` is not a valid format for the `number` type in the OpenAPI spec. I think this should maybe be:
```
- type: integer
  format: int64
```

Reply:
updated